docs/guide.rst

```diff
@@ -1048,38 +1048,35 @@ The following table gives an overview of the available Nox sessions:
    :class: hypermodern-table
    :widths: auto
 
-   ========================================== ================================= ================== =========
-   Session                                    Description                       Python              Default
-   ========================================== ================================= ================== =========
-   :ref:`coverage <The coverage session>`     Report coverage with Coverage.py_ ``3.8``
-   :ref:`docs <The docs session>`             Build Sphinx_ documentation       ``3.8``
-   :ref:`mypy <The mypy session>`             Type-check with mypy_             ``3.6`` … ``3.8``      ✓
-   :ref:`pre-commit <The pre-commit session>` Lint with pre-commit_             ``3.8``                ✓
-   :ref:`safety <The safety session>`         Scan dependencies with Safety_    ``3.8``                ✓
-   :ref:`tests <The tests session>`           Run tests with pytest_            ``3.6`` … ``3.8``      ✓
-   :ref:`typeguard <The typeguard session>`   Type-check with Typeguard_        ``3.6`` … ``3.8``      ✓
-   :ref:`xdoctest <The xdoctest session>`     Run examples with xdoctest_       ``3.6`` … ``3.8``
-   ========================================== ================================= ================== =========
+   ========================================== ===================================== ================== =========
+   Session                                    Description                           Python              Default
+   ========================================== ===================================== ================== =========
+   :ref:`coverage <The coverage session>`     Report coverage with Coverage.py_     ``3.8``
+   :ref:`docs <The docs session>`             Build and serve Sphinx_ documentation ``3.8``
+   :ref:`docs-build <The docs-build session>` Build Sphinx_ documentation           ``3.8``                ✓
+   :ref:`mypy <The mypy session>`             Type-check with mypy_                 ``3.6`` … ``3.8``      ✓
+   :ref:`pre-commit <The pre-commit session>` Lint with pre-commit_                 ``3.8``                ✓
+   :ref:`safety <The safety session>`         Scan dependencies with Safety_        ``3.8``                ✓
+   :ref:`tests <The tests session>`           Run tests with pytest_                ``3.6`` … ``3.8``      ✓
+   :ref:`typeguard <The typeguard session>`   Type-check with Typeguard_            ``3.6`` … ``3.8``      ✓
+   :ref:`xdoctest <The xdoctest session>`     Run examples with xdoctest_           ``3.6`` … ``3.8``
+   ========================================== ===================================== ================== =========
 
 
 .. _The docs session:
 
 The docs session
 ----------------
 
-Build the documentation using the Nox session ``docs``:
+Build and serve the documentation in `interactive mode`__ using the Nox session ``docs``:
```

Comment:
Keep the original simple version:

```suggestion
Build the documentation using the Nox session ``docs``:
```

Keeping it short makes it clear to users that this is what they should be using. The link about Nox's interactive mode does not apply anoymore. The next paragraph explains about the HTTP server, so we can omit that here.